filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -81,20 +84,28 @@ example:
 ----------------------------------------------------------------------
 ./filebeat -e -modules=system,nginx,mysql
```

Comment:
Should we update `-modules` to `--modules`? there are several occurrences on this file